packages/docs/docs/advanced/custom-font.mdx

```diff
@@ -0,0 +1,71 @@
+# Custom font
+
+To use a font from hosters like Google Fonts. First make a css file under `src`
```

Comment:
I'd recommend spliting this into two sections: "Loading fonts from the web" and "Loading fonts locally" by adding the corresponding markdown headers:
```suggestion
## Loading fonts from the web

To use a font from hosters like Google Fonts. First make a css file under `src`
```